backend/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/request/AccountRelationship.java

```diff
@@ -0,0 +1,9 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.request;
+
+import java.util.List;
+
+public class AccountRelationship {
```

Comment:
General comment: what should happen if we receive an unknown field? Right now, it will cause an exception. We could add `@JsonIgnoreProperties(ignoreUnknown = true)` to each pojo to prevent exceptions.

We could even drop all the IT response fields we don't use in notifications. That would make the response pojos simpler.

Reply:
Not sure if this could ever happen. With the request we specify what attributes or kind of attributes we would like to get in the response. I tested several ids and it worked every time. Also the "huge" response of all users are working with the generated response classes.

We can add ignoreUnknown just to be 100% sure.

Reply:
+1 on adding it, we never know when the API will provide new fields

Reply:
done already.